src/software/thunderscope/common/proto_configuration_widget.py

```diff
@@ -37,11 +38,18 @@ def __init__(
         self.on_change_callback = on_change_callback
         self.proto_to_configure = proto_to_configure
 
+        # Create search query bar
+        self.search_query = QLineEdit()
+        self.search_query.textChanged.connect(self.__handle_search_query_changed)
+        self.search_filter_threshold = search_filter_threshold
+
         # Create ParameterGroup from Protobuf
         self.param_group = parametertree.Parameter.create(
             name="params",
             type="group",
-            children=self.config_proto_to_param_dict(self.proto_to_configure, None),
+            children=self.config_proto_to_param_dict(
+                self.proto_to_configure, read_only=False, search_term=None
+            ),
```

Comment:
There's an error here when tscope is launched
```suggestion
            children=self.config_proto_to_param_dict(
                self.proto_to_configure, search_term=None
            ),
```